src/utils/domUtils.ts

```diff
@@ -33,7 +33,7 @@ export function elementBBox(element: SimpleSelection<any>) {
   try {
     bbox = (<any> element.node()).getBBox();
   } catch (err) {
-    bbox = { x: 0, y: 0, width: 0, height: 0 };
+    bbox = { x: 0, y: 0, width: 0, height: 0 } as SVGRect;
```

Comment:
maybe the return type of the function should use `Pick` similar to the other suggestion